chia/data_layer/s3_plugin_service.py

```diff
@@ -56,7 +56,7 @@ async def check_store_id(self, request: web.Request) -> web.Response:
         try:
             data = await request.json()
         except Exception as e:
-            print(f"failed parsing request {request} {e}")
+            log.error(f"failed parsing request {request} {e}")
```

Comment:
Applies to all.  Otherwise you end up with stuff like `2` instead of `KeyError: 2`.

```suggestion
            log.error(f"failed parsing request {request} {type(e).__name__): {e}")
```

Reply:
fixed